packages/core/src/components/progress/progress.story.tsx

```diff
@@ -0,0 +1,36 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+
+export interface ProgressProps extends BaseProps {
+  children?: string | null;
+  'aria-valuetext'?: string;
+}
+
+export const Progress = ({
+  size,
+  value,
+  min = 0,
+  max = 100,
+  hideLabel,
+  children,
+  'aria-valuetext': ariaValuetext,
+  ...props
+}: ProgressProps) =>
+  html('div', {
+    ...props,
+    class: classy(c('progress'), m(size)),
+    role: 'progressbar',
+    'aria-valuenow': String(value),
+    'aria-valuemin': String(min),
+    'aria-valuemax': String(max),
+    'aria-valuetext':
+      ariaValuetext || (typeof children === 'string' ? children : undefined),
+    style: `--percent-value: ${Math.round(
+      ((value - min) * 100) / (max - min)
+    )}%`,
+    children: [
+      !hideLabel
+        ? children || `${Math.round(((value - min) * 100) / (max - min))}%`
```

Comment:
[Minor] Up to you, as this is only for Storybook, however because you do calculation twice maybe you could extract this to a `percentValue` constant and share in these two places.

Reply:
Absolutely! Done.